sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -99,6 +107,9 @@ def __init__(self, **kwargs: Any) -> None:
             # DistributedTracingPolicy(**kwargs),
             config.http_logging_policy or HttpLoggingPolicy(**kwargs)
         ]
+        if self._credential:
+            _add_credential_policy(policies, self._credential)
+
         self.client = AzureMonitorClient(
             host=self._endpoint, connection_timeout=self._timeout, policies=policies, **kwargs)
         self.storage = None
```

Comment:
What is the error message that is displayed when you use and invalid token + when you use a token with the wrong permissions? Specifically the cases covered in [OpenCensus](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py#L134). Are these handled for us or do we need to catch and log?

Reply:
Leaving under HttpResponseError